server/liveblog/embed/embed.py

```diff
@@ -104,6 +105,10 @@ def embed(blog_id, api_host=None, theme=None):
     # retrieve the wanted theme and add it to blog['theme'] if is not the registered one
     try:
         theme_name = request.args.get('theme', theme)
+        theme = get_resource_service('themes').find_one(req=None, name=blog['blog_preferences'].get('theme'))
```

Comment:
You can't because here you are overwriting the `theme` variable given as parameter.
This parameter is useful to overwrite the theme set in the settings.